cli/command/service/opts.go

```diff
@@ -114,6 +124,10 @@ func (i *Uint64Opt) String() string {
 	if i.value != nil {
 		return fmt.Sprintf("%v", *i.value)
 	}
+	if i.hideDefaultValue {
+		// Not display `(default none)`
+		return ""
+	}
```

Comment:
It seems like we'll always want this as the default value. Maybe instead of a flag we should just default to `""` in all cases instead of `"none"`?